src/runtime.js

```diff
@@ -21,6 +21,10 @@ var Compiletime = {
 // code used both at compile time and runtime is defined here, then put on
 // the Runtime object for compile time and support.js for the generated code
 
+function getPointerSize() {
+  return MEMORY64 ? 8 : 4;
+}
```

Comment:
Can this live in parseTools.js with `pointerBits`.. and have the same naming convention?

Also for these function that return constants why not defined constants in parsetools.js?   So `pointerType()` can just be `POINTER_TYPE`?   Not a big deal I guess.

Reply:
Sure, works for me. Would they then be in the global namespace as `POINTER_TYPE` rather than in `Runtime` ?

Reply:
I think the whole `Runtime` namespace is a legacy thing?  Is that right alon?  We should get rid of it I think.     For consistency I think would probably make sense keep `POINTER_TYPE` alongside `POINTER_SIZE` in the `Runtime` object, but I don't feel strongly about that.

Reply:
I've moved most occurences to use the ones in `parseTools.js` instead